api/pkg/apis/v1alpha1/managers/jobs/jobs-manager.go

```diff
@@ -282,16 +282,28 @@ func (s *JobsManager) HandleHeartBeatEvent(ctx context.Context, event v1alpha2.E
 		namespace = "default"
```

Comment:
I think we need add some E2E test cases to check the logic of cleaning up orphaned objects, e.g. re-create object with same name.

Reply:
Added. Wait for test results.